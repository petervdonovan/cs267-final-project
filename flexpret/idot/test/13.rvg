[mu []
[[file {Reset_Handler} {
  Reset_Handler:
    [id init]
    [[single-else get-coreid] t5
      {
        [iterate {address} of [memregion t0 t1 noc-base-address words 2 to 16]
          {
            li t2 42
            sw t2 [id address]
          }]
      }
      {
        [using-noc a3 [mu []
          {
            [delay a4 32]
            [single-hart t1
              {
                [noc-read t0 0]
                [sim-print t0 t3]
              }]
          }]]
      }]
    [id die]
  }]]
]
