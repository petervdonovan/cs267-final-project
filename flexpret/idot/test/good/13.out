Core-1:          0
Core-2:         42
Core-3:         42
Core-4:         42
Core-5:         42
Core-6:         42
Core-7:         42
Core-8:         42
Core-9:         42
Core-10:         42
Core-11:         42
Core-12:         42
Core-13:         42
Core-14:         42
Core-15:         42
Core-0 is done
Core-1 is done
Core-2 is done
Core-3 is done
Core-4 is done
Core-5 is done
Core-6 is done
Core-7 is done
Core-8 is done
Core-9 is done
Core-10 is done
Core-11 is done
Core-12 is done
Core-13 is done
Core-14 is done
Core-15 is done
All cores are done terminating
Assertion failed: Program terminated sucessfully
